zebra-scan/src/service/scan_task/scan.rs

```diff
@@ -325,36 +348,26 @@ pub async fn scan_height_and_store_results(
         // TODO: use the real sapling tree size: `zs::Response::SaplingTree().position() + 1`
         let sapling_tree_size = 1 << 16;
 
-        tokio::task::spawn_blocking(move || {
```

Comment:
Removed multi thread. Same reason as https://github.com/ZcashFoundation/zebra/pull/8567/files#r1613915303 but in this case i was not able to make it work with `spawn_local` so removed all spawn.

Reply:
Fixed in https://github.com/ZcashFoundation/zebra/pull/8568